src/applications/ivc-champva/10-7959a/config/form.js

```diff
@@ -103,12 +110,61 @@ const formConfig = {
       title: 'Sponsor information',
       pages: {
         page2: {
-          path: 'sponsor-information',
+          path: 'sponsor-info',
           title: 'Name',
           ...sponsorNameSchema,
         },
       },
     },
+    beneficiaryInformation: {
```

Comment:
Beneficiary information chapter - includes all the new pages. Main one to pay attention to is the shared address page, where we use a custom page to let users choose another address to use.